### PR TITLE
Add docs on how to run the stdlib unit tests

### PR DESCRIPTION
**Pull request by dusty-phillips**
I thought there might be a script or make target for this, but I didn't see one. The instructions given are what I did to puzzle through it, but I'm not sure they are the best instructions. Happy to improve if there is a better way to do it.

### CODE REVIEW

File: docs/source/guides/contributing.md [hunk 1]
```diff
@@ -108,6 +108,12 @@ fn pub tests(t: mut Tests) {
 When adding a new test module, follow this structure then add it to
 `libstd/test/main.inko`, following the same style as the existing tests.
 
+To run the stdlib tests:
+
+* first build Inko `cargo build release`
+* enter the libstd directory `cd libstd`
+* Run the built inko tests `../target/release/inko test`
```
Comment (yorickpeterse) [inline docs/source/guides/contributing.md:115]: The following steps make this a bit easier:

> 1. Enter the standard library directory using `cd libstd`
> 2. Run the tests using `cargo run -p inko --release -- test`

This way any Rust related changes are picked up right away.

File: docs/source/guides/contributing.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ specific to Inko's version manager should be reported in the project
...
-| inko-lang/inko     | The main project for Inko | <https://github.com/inko-lang/inko/issues>
-| inko-lang/ivm      | Inko's version manager    | <https://github.com/inko-lang/ivm/issues>
-| inko-lang/inko.vim | Vim integration for Inko  | <https://github.com/inko-lang/inko.vim/issues>
-| inko-lang/website  | The Inko website          | <https://github.com/inko-lang/website/issues>
+| Project            | Description               | Issue tracker                                  |
+| :----------------- | :------------------------ | :--------------------------------------------- |
+| inko-lang/inko     | The main project for Inko | <https://github.com/inko-lang/inko/issues>     |
+| inko-lang/ivm      | Inko's version manager    | <https://github.com/inko-lang/ivm/issues>      |
+| inko-lang/inko.vim | Vim integration for Inko  | <https://github.com/inko-lang/inko.vim/issues> |
+| inko-lang/website  | The Inko website          | <https://github.com/inko-lang/website/issues>  |
```
Comment (yorickpeterse) [inline docs/source/guides/contributing.md:25]: These changes are unnecessary, as they're not related to the rest and the trailing `|` isn't needed in Markdown tables.

File: docs/source/guides/contributing.md [hunk 3]
```diff
@@ -56,8 +56,8 @@ how other languages implement the feature (if applicable), and so on.
 ## Submitting changes
 
 !!! note
-    Before submitting code changes, please note that we only accept merge
-    requests for issues labelled as "Accepting contributions".
+Before submitting code changes, please note that we only accept merge
+requests for issues labelled as "Accepting contributions".
```
Comment (yorickpeterse) [inline docs/source/guides/contributing.md:60]: This too isn't necessary, and I think even incorrect given the syntax description at https://squidfunk.github.io/mkdocs-material/reference/admonitions/.

**Comment by yorickpeterse** [comment]
One small note about the commit message: right now the description is aimed towards those reviewing the changes in this PR (e.g. bits such as "Happy to improve if there is a better way to do it."). It's better to describe what is done, what the thought behind it is. In this case the changes are simple enough such that the commit body can just be removed entirely.

**Review by yorickpeterse — CHANGES_REQUESTED** [review]

**Comment by dusty-phillips** [comment]
Oops, sorry about that. Vim auto-formatted the file for me and I forgot to `:noa`.

**Review by yorickpeterse — APPROVED** [review]

**Comment by yorickpeterse** [comment]
@dusty-phillips Thanks! :tada:

**Pull request merged**